README.md

```diff
@@ -23,33 +23,75 @@ where:
 
 ## Configuration file
 
-The configuration file supports the following options:
+The configuration file supports the following top-level options:
 
- * `email = "<address>"` optionally specifies an email address to which any
-   errors running per-repo programs will be sent (warning: full stderr/stdout
-   will be sent, so consider carefully whether these have sensitive information
-   or not).
  * `maxjobs = <int>` is an (optional) non-zero integer specifying the maximum
    number of jobs to run in parallel. Defaults to the number of CPUs in the
    machine.
  * `port = <int>` is a mandatory port number to listen on (e.g. 4567).
+ * `github { ... }` specifies GitHub specific options.
+
+The `github` block supports the following options:
+
  * `reposdir = "<path>"` is the directory where the per-repo programs are
    stored. For a repository `repo` owned by `user` the command
    `<reposdir>/<user>/<repo> <event> <path to GitHub JSON>` will be run. Note
    that per-repo programs are run with their current working directory set to a
    temporary directory to which they can freely write and which will be
    automatically removed when they have completed.
- * `secret = "<secret>"` is the mandatory GitHub secret which guarantees that
-   hooks are coming from your GitHub repository and not a malfeasant.
+
+  * `match "<regex>" { <match-options> }` where `regex` is a regular expression
+    in [Rust regex format](https://docs.rs/regex/1.3.3/regex/index.html) that
```

Comment:
We are linking to a very specific version of the regex crate docs here. I wonder if we should link to somepleace that will move with the times?

Reply:
Good point. Fixed in https://github.com/softdevteam/snare/pull/13/commits/18c83e10a049c2f1c5cd76fb0dfe7032d3ca22a4.